src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -5,30 +5,16 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+	public List<Long> findHappyNumbersInRange(long start, long end) {
 
 		List<Long> result = new ArrayList<Long>();
 
-		for (long i = a; i < b; i++) {
+		for (long happyNumberCandidate = start; happyNumberCandidate < end; happyNumberCandidate++) {
 
-			List<Long> longs = new ArrayList<Long>();
+			long sum = doHappyNumberCalculation(happyNumberCandidate);
```

Comment:
The result of this is not necessarily a "Happy number". So the function name should be different

Reply:
Suggestion: sum = sumOfPowers(...)